packages/ui/src/1_atoms/Icon/Icon.tsx

```diff
@@ -76,6 +76,7 @@ export const Icon: React.FC<IconProps> = ({
         width={iconSize}
         fill="currentColor"
         className={classNames(className, inlineBlock)}
+        data-icon={icon}
```

Comment:
I know it's not specifically mentioned in the description but we had a discussion with @soulBit and we want to have `data-action-id` in all of the components so pls add it to this component as well (take a look at `Input` and `InputBase` in https://github.com/DistributedCollective/sovryn-dapp/pull/8 ).